Crypto/Tests/Data4LifeCryptoProtocolTests.swift

```diff
@@ -63,7 +63,7 @@ class Data4LifeCryptoProtocolTests: XCTestCase {
     func testEncryptTags() {
         do {
             let testModel: SymCryptoTestModel = try bundle.decodable(fromJSON: "symTagEncrypt")
-            let blankIV = [UInt8](repeating: 0x00, count: testModel.key.ivSize).asData
+            let blankIV = [UInt8](repeating: 0x00, count: testModel.key.defaultIvSize).asData
```

Comment:
Sidequestion: Should the IV not be filled with random bytes?...since this is "just" a test case, I'd like to asks, if it is done in the production code.